README.md

```diff
@@ -1142,6 +1159,14 @@ Currently being reworked as Neo Launcher.
 - [x] [GitHub](https://github.com/kigya/NotedgeApp)
 - [x] [Official page](https://notedge.netlify.app)
 
+### Joplin
```

Comment:
Doesn't this app needed an account or I have a bad memory?

Reply:
Account feature is completely optional, and it also supports notes to be saved on file system which then could be synced anyhow user want.